skyreader/utils/handle_map_data.py

```diff
@@ -400,8 +400,10 @@ def prepare_multiorder_map(
         max_nside = np.max(all_nsides)
         multiorder_map = mhealpy.HealpixMap(
             grid_value / healpy.nside2pixarea(
-                max_nside, degrees=True,
-            ), uniq_array
+                max_nside, degrees=False,
+            ),
+            uniq_array,
+            unit="sr-1"
         )
-        column_names = [f"{column_names[0]} DENSITY [deg-2]"]
+        column_names = ["PROBDENSITY"]
```

Comment:
If you want, you can be fancy: define an `enum.StrEnum` class for titles, then use class members where needed. Personally, I'd leave this as is, but it's worth pointing out this option.

Reply:
`enum.StrEnum` is only for 3.11+, so ignore this suggestion :smile: